.github/actions/sync/sync.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+ls -all
```

Comment:
```suggestion
ls -al
```

The last `l` is a repetition.